cpp/tests/test_graph_simulation.cpp

```diff
@@ -201,18 +202,39 @@ TEST(TestGraphSimulation, consistencyStochasticMobility)
     g.add_edge(0, 1, Eigen::VectorXd::Constant(4, 0.001));
 
     auto sim = mio::make_migration_sim(t0, dt, std::move(g));
-    
-    //set seeds
-    sim.get_rng().seed({114381446, 2427727386, 806223567, 832414962, 4121923627, 1581162203});
+
+    ScopedMockDistribution<testing::StrictMock<MockDistribution<mio::ExponentialDistribution<ScalarType>>>>
+        mock_exponential_dist;
+    // use pregenerated exp(1) random values
+    EXPECT_CALL(mock_exponential_dist.get_mock(), invoke)
+        .Times(testing::Exactly(10))
+        .WillOnce(testing::Return(0.446415))
+        .WillOnce(testing::Return(1.04048))
+        .WillOnce(testing::Return(0.136687))
+        .WillOnce(testing::Return(2.50697))
+        .WillOnce(testing::Return(1.61943))
+        .WillOnce(testing::Return(0.267578))
+        .WillOnce(testing::Return(1.03696))
+        .WillOnce(testing::Return(0.58395))
+        .WillOnce(testing::Return(0.113943))
+        .WillOnce(testing::Return(1.204045));
```

Comment:
maybe a comment what effect each value has/should have (one comment for each value if necessary, one comment at the top otherwise)

Reply:
I added one comment for all values, since they are all assigned to the same variable, at two places in advance.